config/default/UserLanguageService.conf.php

```diff
@@ -4,4 +4,5 @@
 
 return new UserLanguageService([
     UserLanguageService::OPTION_LOCK_DATA_LANGUAGE => true,
+    UserLanguageService::OPTION_DEFAULT_LANGUAGE => 'nb-NO',
```

Comment:
This will work for new instance (it's ok), but also you need to create a migration, which will override the current config with the new one.

Reply:
I think this should work: 8de93a690bb8bf58f37f15c2340caec6509181b7. @shpran can you please take a quick look and check that is OK? :)

Reply:
I am not sure we should call it `DEFAULT_INTERFACE_LANGUAGE` now 🤔

Reply:
I'll rename it ro `OPTION_INTERFACE_LANGUAGE` / `interface_language`.

Would it be better to drop it altogether from this file (Generis) and just keep the migrations that sets its value to nb-NO in the client extension?

Reply:
Renamed as interface_language in bf796f4c79b697834f18acffdea978ffc350249d

Reply:
This .conf is not being modified by this pull request anymore (we are modifying it from client-specific migrations instead). Therefore I'm closing this thread.